pkg/plugins/plugins.go

```diff
@@ -164,11 +164,14 @@ func scan(pluginDir string) error {
 }
 
 func (scanner *PluginScanner) walker(currentPath string, f os.FileInfo, err error) error {
+	// TODO: Based on how we search for plugin.json in grafan-net fetchPluginJson, it should be enough to scan
+	// just /src, /dist or / paths inside the plugin dir
+
 	if err != nil {
 		return err
 	}
 
-	if f.Name() == "node_modules" {
+	if f.Name() == "node_modules" || f.Name() == "Chromium.app" {
```

Comment:
Not liking this hardcoding here like this. If we allowed just one level of traversal it could probably just go away.